web/public/static/langs/en.json

```diff
@@ -171,7 +171,7 @@
   "subscribe_dialog_subscribe_description": "Topics may not be password-protected, so choose a name that's not easy to guess. Once subscribed, you can PUT/POST notifications.",
   "subscribe_dialog_subscribe_topic_placeholder": "Topic name, e.g. phil_alerts",
   "subscribe_dialog_subscribe_use_another_label": "Use another server",
-  "subscribe_dialog_subscribe_use_another_background_info": "Note: Background notifications are not supported on other servers",
+  "subscribe_dialog_subscribe_use_another_background_info": "Note: Notifications from other servers will not be received when the web app is not open",
```

Comment:
Can you drop the `Note:`? Otherwise I'm good with this.

Reply:
Done!